File: PROGRAMMERS/lv0/javascript/옷가게 할인 받기.js
function solution(price) {
  let dis = [
    [500_000, 0.8],
    [300_000, 0.9],
    [100_000, 0.95],
  ];

  for (let [s, p] of dis) {
    if (price >= s) {
      return parseInt(price * p);
    }
  }
  return price;
}

let inps = [150_000, 580_000];

for (let i of inps) {
  console.log(solution(i));
}

File: PROGRAMMERS/lv0/python/옷가게 할인 받기.py
def solution(price):
    dis = [[500_000, 0.8], [300_000, 0.9], [100_000, 0.95]]
    for (v, r) in dis:
        if(price >= v): return int(price*r)
    return price

inps = [
    150_000,
    580_000
]

for inp in inps:
    print(solution(inp))